extensions/iceberg/src/main/java/io/deephaven/iceberg/location/IcebergTableParquetLocationKey.java

```diff
@@ -0,0 +1,51 @@
+//
+// Copyright (c) 2016-2024 Deephaven Data Labs and Patent Pending
+//
+package io.deephaven.iceberg.location;
+
+import io.deephaven.engine.table.impl.locations.TableLocationKey;
+import io.deephaven.parquet.table.ParquetInstructions;
+import io.deephaven.parquet.table.location.ParquetTableLocationKey;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+import java.net.URI;
+import java.util.Map;
+
+/**
+ * {@link TableLocationKey} implementation for use with data stored in Iceberg tables in the parquet format.
+ */
+public class IcebergTableParquetLocationKey extends ParquetTableLocationKey implements IcebergTableLocationKey {
+
+    private static final String IMPLEMENTATION_NAME = IcebergTableParquetLocationKey.class.getSimpleName();
+    private final ParquetInstructions readInstructions;
```

Comment:
You don't need this. Just make the parent's reference `protected`, or add an accessor to the parent.